go-controller/pkg/node/gateway_udn.go

```diff
@@ -157,12 +178,15 @@ func (b *bridgeConfiguration) patchedNetConfigs() []*bridgeUDNConfiguration {
 // bridgeUDNConfiguration holds the patchport and ctMark
 // information for a given network
 type bridgeUDNConfiguration struct {
-	patchPort   string
-	ofPortPatch string
-	masqCTMark  string
-	pktMark     string
-	v4MasqIPs   *udn.MasqueradeIPs
-	v6MasqIPs   *udn.MasqueradeIPs
+	patchPort              string
+	ofPortPatch            string
+	masqCTMark             string
+	pktMark                string
+	v4MasqIPs              *udn.MasqueradeIPs
+	v6MasqIPs              *udn.MasqueradeIPs
+	subnets                []config.CIDRNetworkEntry
+	nodeSubnet             []*net.IPNet
+	isUDNNetworkAdvertised bool
```

Comment:
nit: s/isUDNNetworkAdvertised/isNetworkAdvertised

Reply:
Also, we probably need this to be an atomic, since `isUDNNetworkAdvertised` can change at any time and you will get a race